content/en/about/bundle.md

```diff
@@ -0,0 +1,298 @@
+# Sigstore Bundle
+
+October 31, 2024
+
+Version 0.3.2
+
+This document describes the data structure for storing Sigstore signatures generated by tooling
+working in the context of the Sigstore Public Instance. It includes `json` examples of
+serialized bundles of the current bundle format version. It may exclude descriptions of parameters
+that continue to exist for compatibility reasons or for private use cases. For a full description of
+the format, the formal schema and information about language library support see
+[sigstore/protobuf-specs](https://github.com/sigstore/protobuf-specs).
+
+## Bundle
+
+A Sigstore bundle is everything required to verify a signature on an artifact. This
+is satisfied by the **Verification Material** and signature **Content**.
+
+### Verification Material
+
+This is key materical used to verify signatures along with supporting metadata like transparency log entries and timestamps. Bundles must include at least one transpareny log's signed entry timestamp or an rfc3161 timestamp to provide proof of signing time.
+
+#### Key Material
+
+A single X.509 leaf certificate conveying the signing key and containing [extensions](https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md) 
+for identities consumed at verification time. This is the recommended `"verificationMaterial"` type
+for use with the public Sigstore infrastructure.
+
+```json
+"verificationMaterial": {
+  "certificate": {
+    "rawBytes": "<Base64(DER bytes)>"
+  }
+}
+```
+
+#### Transparency Log Entries
+
+One or more transparency logs entries to provide proof of inclusion in a public log and optionally a timestamp to
+validate signing occurred at a valid point in time.
```

Comment:
I'd define what we mean by "valid point in time", as in a user was in possession of a signing key and used it to produce a signature, and that signing key was certified by a CA, and this is recorded while the certificate is valid in the log.

Reply:
done